src/util/strftime.ts

```diff
@@ -140,7 +140,12 @@ const formatCodes = {
 };
 (formatCodes as any).h = formatCodes.b
 
-export default function (d: Date, formatStr: string) {
+export default function (inputDate: Date, formatStr: string) {
+  let d = inputDate
+  if (d instanceof TimezoneDate && d.timezoneOffset !== null) {
```

Comment:
Add `new Date().getTimezoneOffset()` to `d.timezoneOffset` to eliminate the `!== null` test. It also means if d is an instance of TimezoneData, then it's always available.

Reply:
I eliminated the null test but made it default to zero instead of `getTimezoneOffset()` so that it still works with timezoneless inputs